lib/absinthe/type/built_ins/scalars.ex

```diff
@@ -59,6 +59,48 @@ defmodule Absinthe.Type.BuiltIns.Scalars do
     parse parse_with([Absinthe.Blueprint.Input.Boolean], &parse_boolean/1)
   end
 
+  scalar :utc_datetime, name: "DateTime" do
```

Comment:
I was following the same conventions than Ecto (which I concede might be driven by legacy). I was also considering what an API consumer would see during introspection and I thought this choice would make the most sense. That said being more specific doesn't hurt :)